Makefile

```diff
@@ -68,23 +68,7 @@ test:
 bench:
 	go test -timeout=30s -bench=. $$(go list ./...)
 
-check: gofmt ineffassign lint staticcheck vet
+check:
+	docker run --rm -v `pwd`:/app -w /app docker.io/golangci/golangci-lint:v1.40.1 golangci-lint run -v
```

Comment:
How does it pick the same list of plugins as the ones configured in the CI job?

Reply:
By mounting pwd onto `/app` and setting `/app` as cwd, it should use the ` .golangci.yml` file added in this PR.